packages/core/package.json

```diff
@@ -44,7 +44,7 @@
     "prettier": "^3.2.5",
     "rollup-plugin-dts": "^6.1.0",
     "tslib": "^2.6.2",
-    "typescript": "^5.4.5"
+    "typescript": "^5.1.6"
```

Comment:
What is the reason for downgrading typescript?

Reply:
To get the support for eslint. Eslint was throwing some errors and was not able to lint some files. Downgraded to the latest stable version that eslint supports.